cypress/utils/utils.ts

```diff
@@ -126,7 +126,25 @@ export function click(fieldId: string, isForced = true, log = false): void {
     if (!log) {
         cy.log(`Click ${fieldId}`);
     }
-    cy.get(fieldId, { log, timeout: 60 * SEC }).click({ log, force: isForced });
+    cy.get(fieldId, { log, timeout: 30 * SEC }).click({ log, force: isForced });
+}
+
+export function clickWithFocus(fieldId: string, isForced = true, log = false): void {
+    if (!log) {
+        cy.log(`Click ${fieldId}`);
+    }
+    cy.get(fieldId, { log, timeout: 30 * SEC })
+        .focus()
+        .click({ log, force: isForced });
+}
+
+export function clickJs(fieldId: string, isForced = true, log = false): void {
```

Comment:
@ibragins , We already have the click() fn in utils.ts. I'd like to make sure I'm not missing something. How's clickJs() different from the existing click() ?

Reply:
It utilizes natural JS `.click()` function instead of using Cypress' functionality. We need this because sometimes original Cypress function doesn't work.